modules/assignation.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, bot):
     async def assign(ctx, assignee: Option(discord.Member, "Assigns your role to someone", required=False) = None):
         """Assigns your role to you"""
 
-        maaldar_user = check_if_user_exists(Assignation.cursor, ctx.author.id)
+        maaldar_user = check_if_user_exists(ctx.author.id)
```

Comment:
since Assignation.cursor is redundant, the class shouldnt have any connection and cursor either